core/stats.go

```diff
@@ -78,3 +78,22 @@ func (ds *dockerService) ListContainerStats(
 
 	return &runtimeapi.ListContainerStatsResponse{Stats: stats}, nil
 }
+
+// PodSandboxStats returns stats for a pod sandbox stats request
+func (ds *dockerService) PodSandboxStats(
+	_ context.Context,
+	r *runtimeapi.PodSandboxStatsRequest,
+) (*runtimeapi.PodSandboxStatsResponse, error) {
+	// TODO
```

Comment:
Can we implement this `TODO` since it is done in the alpha version? Or are the same methods not available?

Reply:
The alpha just "translates" the API between v1 and v1alpha2. It is the same (unimplemented) methods.